FAQ.md

```diff
@@ -302,6 +303,37 @@ Set `DEVITO_IGNORE_UNKNOWN_PARAMS=1` to avoid Devito raising an exception if one
 [top](#Frequently-Asked-Questions)
 
 
+## What are the accepted combinations of PLATFORM, ARCH, and LANGUAGE
+
+#### LANGUAGE={C,openmp}
+
+These two languages can be used with virtually any PLATFORM and ARCH.
+
+With a device PLATFORM (e.g., `nvidiaX` or `amdgpuX`), the compiler will generate OpenMP code for device offloading.
+
+When using OpenMP offloading, it is however recommended to stick to the corresponding vendor compiler, so `ARCH=amdclang` for AMD, `ARCH={icc,icx,intel}` for Intel, and `ARCH=nvc` for NVidia.
```

Comment:
Maybe note that icx is the default when using intel?
and ICC gonna be deprecated?